qemu/tests/hugepage_mem_stress.py

```diff
@@ -35,6 +37,12 @@ def heavyload_install():
     vm.verify_alive()
     session = vm.wait_for_login()
 
+    if params.get("extra_params") == " -mem-path /mnt/hugepage":
```

Comment:
I suggest to check if the path is mounted with hugepage here.

Reply:
When env_process, the HugePageConfig.setup() will cache is cleared and large memory pages are specified. mount to / mnt / kvm_hugepage. You can see this in the log.
04:44:51 DEBUG| Amount of memory used by each vm: 117760
04:44:51 DEBUG| System setting for large memory page size: 2048
04:44:51 DEBUG| Number of large memory pages needed for this test: 58944
04:44:52 DEBUG| Successfully set 58944 large memory pages on host 
04:44:52 INFO | Running 'mount -t hugetlbfs -o pagesize=2048K none /mnt/kvm_hugepage'

So I think this is enough to show that /mnt/hugepage  is no mount hugepages.
And in the following checkpoint, it will also prove that HugePages on the host are not used. Case fail if used.

Reply:
I made the suggestion because this line looks like hard code to me. Besides, the log shows  /mnt/kvm_hugepage is mounted, but nothing about /mnt/hugepage.

Reply:
correct